### ISSUE TITLE
Exception occurred Some of the required values for successive user emulation are missing

### ISSUE THREAD

**Issue by Naoray** [igbt6/Packt-Publishing-Free-Learning#86]
Just logged into my server and noticed the script couldn't claim an ebook since october 20th.

```
[INFO] - Creating session...
[INFO] - Session created, logged in successfully!
[INFO] - Start grabbing eBook...
[INFO] - Captcha detected. Trying to solve it using Anti-captcha.com.
[ERROR] - Exception occurred Some of the required values for successive user emulation are missing.. Website stoken is invalid. Expected string value, received array
```

Does someone know how to solve this? Am I the only one facing this issue?

**Comment by ad-m** [comment]
Yes, [duplicate](https://github.com/igbt6/Packt-Publishing-Free-Learning/issues?q=is%3Aissue+%22%27Some+of+the+required+values+for+successive%22+is%3Aclosed).

tl;dr: ```pip install python-anticaptcha -U```
